### PR TITLE
fix: corrected heading of ReadAll

### LINKED ISSUE #107: Title mismatches at readAll activity
**Describe the bug**
In ReadAll activity, while fetching data if we want to fetch both paid and not paid so while showing list view heading does not match

**To Reproduce**
1. Go to ReadAll activity
2. Click on both paid and not paid and readAll
3. observe at the top 

**Smartphone (please complete the following information):**
 - Device: Samsung j8
 - OS: Android 9 Pie

**Would you like to work on the issue?** (Yes/No)
No

### PR DESCRIPTION
**Pull request by hridyakrishna**
Fixed #107 

Changes: Changed the label of the activity in AndroidManifest.xml.
                 Changed the heading of ReadAll when multiple checkboxes are ticked (ReadAllData.java)

Screenshots of the change: 
photo_2019-08-15_13-48-56

### CODE REVIEW

File: app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java [hunk 1]
```diff
@@ -78,10 +78,15 @@ public void poojaNotPaid(View view) {
         public void readAllButton(View view) {
             if (readAllBinding.donatePaid.isChecked() && readAllBinding.donateNotPaid.isChecked()) {
                 flag = 5;
+                readAllBinding.heading.setText(
```
Comment (Chromicle) [inline app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java]: use string resources

File: app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java [hunk 2]
```diff
@@ -78,10 +78,15 @@ public void poojaNotPaid(View view) {
...
+                        "List of Donated Poojas:Both paid and not paid");
             }
 
             if (readAllBinding.poojaPaid.isChecked() && readAllBinding.poojaNotPaid.isChecked()) {
                 flag = 6;
+                readAllBinding.heading.setText("List of registered Poojas:Both paid and not paid"
```
Comment (Chromicle) [inline app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java]: same as above

File: app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java [hunk 3]
```diff
@@ -78,10 +78,12 @@ public void poojaNotPaid(View view) {
         public void readAllButton(View view) {
             if (readAllBinding.donatePaid.isChecked() && readAllBinding.donateNotPaid.isChecked()) {
                 flag = 5;
+                readAllBinding.heading.setText("List of Donated Poojas:Both paid and not paid");
```
Comment (yashk2000) [inline app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java]: Please use string resources.

File: app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java [hunk 4]
```diff
@@ -78,10 +78,12 @@ public void poojaNotPaid(View view) {
...
             }
 
             if (readAllBinding.poojaPaid.isChecked() && readAllBinding.poojaNotPaid.isChecked()) {
                 flag = 6;
+                readAllBinding.heading.setText("List of registered Poojas:Both paid and not paid");
```
Comment (yashk2000) [inline app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java]: String resources.

**Pull request merged**